bindings/matrix-sdk-crypto-js/src/verification.rs

```diff
@@ -1073,6 +1084,8 @@ impl VerificationRequest {
     }
 
     /// Transition from this verification request into a SAS verification flow.
+    ///
+    /// Returns `Promise<[Sas, RoomMessageRequest|ToDeviceRequest] | undefined>`
```

Comment:
oh, this is an unrelated thing that was apparently sitting in my working copy. It's pretty trivial so I'm inclined to leave it in this PR.

Reply:
Yup, agreed.